client/gameObjects/Skeleton.js

```diff
@@ -119,6 +120,21 @@ export default class Skeleton extends Phaser.GameObjects.Image {
       }
     }
 
+    if (destTileX && destTileY) {
+      if (this.markedDestTile) {
+        this.markedDestTile.setAlpha(1);
+        this.markedDestTile = null;
+      }
+
+      const tile = this.scene.groundLayer.getTileAt(destTileX, destTileY);
+
+      tile.setAlpha(0.6);
+      this.markedDestTile = tile;
+    } else if (this.markedDestTile) {
+      this.markedDestTile.setAlpha(1);
+      this.markedDestTile = null;
+    }
```

Comment:
`this.markedDestTile` is always cleared, move it out of `if` block:
```suggestion
    if (this.markedDestTile) {
      this.markedDestTile.clearAlpha();
      this.markedDestTile = null;
    }
    if (destTileX && destTileY) {
      const tile = this.scene.groundLayer.getTileAt(destTileX, destTileY);
      tile.setAlpha(0.6);
      this.markedDestTile = tile;
    }
```